### ISSUE TITLE
Bug: exception raised when `.guess_media(...)` returns no results

### ISSUE THREAD

**Issue by LovecraftianHorror** [LovecraftianHorror/subwinder#46]
## Bug Report

* **`subwinder` version:** `current master` (131e210b36e6777194853824fff5629a9f96e5a6)
* **OS name and version:** N/A

### Description

So I've seen this a couple of ways so far. On an empty string (`""`) `GuessMovie` returns an empty list resulting in a `TypeError` when a value is attempted to be returned by a key. The other is when there is a string that just doesn't return any matches (`"asjdlfkjweifadsfjk"`). In this case, it returns a pretty bare `dict` (see below). Both of these will effect both the ranked and unranked variants. I suppose the best option from here would be to return `None` for either one since there isn't enough information for a `GuessMediaResult` or any `MediaInfo` derivatives. The other option would be to return `GuessMediaResult` for the unranked variant that has no information (full of empty lists and `None`, which might be better since stuff will likely be expecting a `GuessMediaResult` and should be able to handle a lack of information.

```json
{
    "asjdlfkjweifadsfjk": {
        "GuessIt": {
            "title": "asjdlfkjweifadsfjk",
            "type": "movie"
        },
        "GetIMDBSuggest": []
    }
}
```

### Code to reproduce

```python
from subwinder import AuthSubwinder

with AuthSubwinder() as asw:
    asw.guess_media([""])
    asw.guess_media(["asjdlfkjweifadsfjk"])
```

**Comment by LovecraftianHorror** [comment]
To make this more complicated it looks like `""` is actually just exluded from the results entirely, so we have to go through and check for it and add in an empty value explicitly.

**Comment by LovecraftianHorror** [comment]
Fixed in c2eed989b30a44026b974d13956cb0ec721927cd.
